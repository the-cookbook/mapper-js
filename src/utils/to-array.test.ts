import toArray from './to-array';

describe('utils/toArray()', () => {
  it('should match snapshot', () => {
    expect(toArray).toMatchSnapshot();
  });

  it('should parse type to array', () => {
    const source = 'lorem ipsum';
    const expected = [source];

    expect(toArray(source)).toStrictEqual(expected);
  });

  it('should not parse if type is already an array', () => {
    const source = ['lorem ipsum'];
    const expected = source;

    expect(toArray(source)).toStrictEqual(expected);
  });
});
