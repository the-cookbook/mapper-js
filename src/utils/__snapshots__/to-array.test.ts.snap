// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils/toArray() should match snapshot 1`] = `[Function]`;
